views/styles/login_page_styles.js

```diff
@@ -0,0 +1,69 @@
+import { colors } from '../../constants'
+
+export const loginStyles = (theme) => ({
+  login_container: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignContent: 'center',
+    height: '100vh',
+    width: '100vw',
+    alignItems: 'center',
+    backgroundColor: '#f9f9f9',
```

Comment:
When we start getting the real designs in place I think it would be nice to move things like colors into constants. Not much point right now though.

Reply:
good catch we do have a colors constant